### PR TITLE
Add 2.3.0 blog post

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

**Comment by potiuk** [comment]
🎉 🤞

File: landing-pages/site/content/en/blog/airflow-2.3.0/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+date: "TBD"
+---
+
+Apache Airflow 2.3.0 contains over TBD commits since 2.2.0 and includes TBD new features, TBD improvements, TBD bug fixes, and several doc changes.
+
+**Details**:
+
+📦 PyPI: https://pypi.org/project/apache-airflow/2.3.0/ \
+📚 Docs: https://airflow.apache.org/docs/apache-airflow/2.3.0/ \
+🛠️ Changelog: https://airflow.apache.org/docs/apache-airflow/2.3.0/changelog.html \
```
Comment (jedcunningham) [inline landing-pages/site/content/en/blog/airflow-2.3.0/index.md:18]:
```suggestion
🛠️ Release Notes: https://airflow.apache.org/docs/apache-airflow/2.3.0/release_notes.html \
```

File: landing-pages/site/content/en/blog/airflow-2.3.0/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+@task
+def consumer(arg):
+    print(list(arg))
+
+
+with DAG(dag_id="dynamic-map", start_date=datetime(2022, 4, 2)) as dag:
+    consumer.expand(arg=make_list())
+```
+
+More information can be found here: [Dynamic Task Mapping](https://airflow.apache.org/docs/apache-airflow/latest/concepts/dynamic-task-mapping.html)
```
Comment (jedcunningham) [inline landing-pages/site/content/en/blog/airflow-2.3.0/index.md:50]:
```suggestion
More information can be found here: [Dynamic Task Mapping](https://airflow.apache.org/docs/apache-airflow/2.3.0/concepts/dynamic-task-mapping.html)
```

Probably better to link to the specific version? Or it at least should be to `stable`.

File: landing-pages/site/content/en/blog/airflow-2.3.0/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+Grid view replaces tree view in Airflow 2.3.0.
+
+**Screenshots**:
+![The new grid view](grid-view.png)
+
+
+## LocalKubernetesExecutor
+
+Airflow 2.3.0, features a new executor called LocalKubernetesExecutor. This executor helps you run some tasks using LocalExecutor and run another set of tasks using the KubernetesExecutor in the same deployment using task's queue to coordinate the switching.
```
Comment (jedcunningham) [inline landing-pages/site/content/en/blog/airflow-2.3.0/index.md:62]:
```suggestion
There is a new executor names LocalKubernetesExecutor. This executor helps you run some tasks using LocalExecutor and run another set of tasks using the KubernetesExecutor in the same deployment based on the task's queue.
```

nit

File: landing-pages/site/content/en/blog/airflow-2.3.0/index.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+More information can be found here: [LocalKubernetesExecutor](https://airflow.apache.org/docs/apache-airflow/latest/executor/local_kubernetes.html)
+
+
+## DagProcessorManager as standalone process (AIP-43)
+
+As of 2.3.0, you can run the DagProcessorManager as a standalone process. Because DagProcessorManager runs user code, separating it from the scheduler process and running it as an independent process in a different host is a good idea.
+
+`airflow dag-processor` cli command will start a new process that will run the DagProcessorManager in a separate process. Before you can run the DagProcessorManager as a standalone process, you need to set the `AIRFLOW__SCHEDULER__STANDALONE_DAG_PROCESSOR` to `True`.
```
Comment (jedcunningham) [inline landing-pages/site/content/en/blog/airflow-2.3.0/index.md:71]:
```suggestion
The `airflow dag-processor` cli command will start a new process that will run the DagProcessorManager in a separate process. Before you can run the DagProcessorManager as a standalone process, you need to set the [[scheduler] standalone_dag_processor](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#standalone_dag_processor) to `True`.
```

File: landing-pages/site/content/en/blog/airflow-2.3.0/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+```
+You can also use `json` serialization format when setting the connection in environment variables.
+
+More information can be found here: [JSON serialization for connections](https://airflow.apache.org/docs/apache-airflow/latest/howto/connection.html)
+
+## Airflow `db downgrade` and Offline generation of SQL scripts
+
+Airflow 2.3.0 introduced a new command `airflow db downgrade` that will downgrade the database to your chosen version.
+
+You can also generate the downgrade/upgrade SQL scripts for your database and manually run it against your database or just view the SQL scripts that would be run by the downgrade/upgrade command.
```
Comment (jedcunningham) [inline landing-pages/site/content/en/blog/airflow-2.3.0/index.md:101]:
```suggestion
You can also generate the downgrade/upgrade SQL scripts for your database and manually run it against your database or just view the SQL queries that would be run by the downgrade/upgrade command.
```

File: landing-pages/site/content/en/blog/airflow-2.3.0/index.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+More information can be found here: [Reuse of decorated DAGs](https://airflow.apache.org/docs/apache-airflow/latest/tutorial_taskflow_api.html#reusing-a-decorated-task)
+
+## Other small features
+
+This isn’t a comprehensive list, but some noteworthy or interesting small features include:
+
+- Support different timeout value for dag file parsing
+- `airflow dags reserialize` command to reserialize dags
+- `db clean` CLI command for purging old data
```
Comment (jedcunningham) [inline landing-pages/site/content/en/blog/airflow-2.3.0/index.md:133]: I think this should get a full section!

File: landing-pages/site/content/en/blog/airflow-2.3.0/index.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+This isn’t a comprehensive list, but some noteworthy or interesting small features include:
+
+- Support different timeout value for dag file parsing
+- `airflow dags reserialize` command to reserialize dags
+- Events Timetable
+- SmoothOperator - Operator that does literally nothing except logging a YouTube link to
+    Sade's "Smooth Operator". Enjoy!
+
+## Contributors
+Thanks to everyone who contributed to this release: TBD
```
Comment (kaxil) [inline landing-pages/site/content/en/blog/airflow-2.3.0/index.md:145]:
```suggestion
Thanks to everyone who contributed to this release: Ash Berlin-Taylor, Brent Bovenzi, Daniel Standish, Elad, Ephraim Anierobi, Jarek Potiuk, Jed Cunningham, Josh Fell, Kamil Breguła, Kanthi, Kaxil Naik, Khalid Mammadov, Malthe Borch, Ping Zhang, Tzu-ping Chung and many others who keep making Airflow better for everyone.
```

This was based on "git shotlog" and top 15 contributors on the diff of 2.2.5 and 2.3.0

**Review by kaxil — APPROVED** [review]

**Review by jedcunningham — APPROVED** [review]

**Review by kaxil — COMMENTED** [review]
hmm the blog post isn't visible on the site for some reason, throws a 404 for me

**Comment by jedcunningham** [comment]
It was the date. Fixed, it's up.

**Pull request merged**